### PR TITLE
Add Swift frameworks

### PR DESCRIPTION
**Pull request by kiliankoe**
Hope you don't mind I just threw all three together into a single PR. Vapor, Perfect and Kitura. All build and run fine. 

I also added make and neph build targets and threw everything into the benchmarker as well.

### CODE REVIEW

**Comment by tbrand** [comment]
> Hope you don't mind I just threw all three together into a single PR

NP :smile:
I'll review this, thanks!!

File: swift/vapor/Sources/main.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Vapor
+
+let drop = Droplet(environment: .production)
```
Comment (kiliankoe) [inline swift/vapor/Sources/main.swift]: On looking over this again I'm pretty sure stating the environment here is unnecessary. As far as I can tell this only applies to multiple config environments being present.

Comment (kiliankoe) [inline swift/vapor/Sources/main.swift]: Nevermind, without this vapor falls back to default logging, which it shouldn't for this use case.

**Comment by tbrand** [comment]
It works very fine!!! Perfect!!!

**Comment by kiliankoe** [comment]
Thanks 😊

**Pull request merged**